<launch>

  <param name="model_server_url" value="$(env MODEL_SERVER_URL)" />

  <include file="$(find astra_launch)/launch/astra.launch" />

  <node name="ngrok" pkg="image_caption_machine" type="start_ngrok.sh" />

  <node name="alexa_server" pkg="image_caption_machine" type="alexa_server.py" />
  <node name="caption_service" pkg="image_caption_machine" type="caption_service.py" />
  <node name="caption_client" pkg="image_caption_machine" type="caption_client.py" />

</launch>
